Importing certain code fails, CFG post-order panic

The following program `sample.go` triggers a panic:

```go
import "github.com/seeruk/go-console"
```

(You'd need to pull that in as a dependency, and vendor it. I'm using the tag v0.1.0-alpha.5 currently. It's a project of my own).

Expected result:
```console
$ yaegi
> import "github.com/seeruk/go-console"
0x__________
> 
```

Got:
```console
$ yaegi
> import "github.com/seeruk/go-console"
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: /Users/seer/GoProjects/pkg/mod/github.com/seeruk/go-console@v0.1.0-alpha.5/specification/argument_parser.go:38:15: CFG post-order panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1245073]

goroutine 36 [running]:
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2.1(0xc0002c3300)
	/Users/seer/git/github.com/containous/yaegi/interp/cfg.go:365 +0xba
panic(0x18e7d20, 0x212c950)
	/usr/local/Cellar/go/1.13.4/libexec/src/runtime/panic.go:679 +0x1b2
github.com/containous/yaegi/interp.(*itype).finalize(0xc0003a5930, 0xcb4ab4367a83a0e5, 0xc0001ddf10, 0xcb0000000123bb15)
	/Users/seer/git/github.com/containous/yaegi/interp/type.go:631 +0x1b3
github.com/containous/yaegi/interp.(*itype).frameType(0xc00035be10, 0x12061f4, 0x18e71e0)
	/Users/seer/git/github.com/containous/yaegi/interp/type.go:880 +0x2f
github.com/containous/yaegi/interp.(*scope).add(0xc000444e40, 0xc00035be10, 0xc000510b00)
	/Users/seer/git/github.com/containous/yaegi/interp/scope.go:151 +0x49
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc0002c3300)
	/Users/seer/git/github.com/containous/yaegi/interp/cfg.go:773 +0x95c5
github.com/containous/yaegi/interp.(*node).Walk(0xc0002c3300, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:171 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc0002c3000, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002c2f00, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002c2200, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002c1200, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0002ac500, 0xc0001dea90, 0xc0001deac8)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc00026e2c0, 0xc0002ac500, 0xc000516540, 0x2a, 0x0, 0x0, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/cfg.go:47 +0x1b2
github.com/containous/yaegi/interp.(*Interpreter).importSrc(0xc00026e2c0, 0x0, 0x0, 0xc0005160f1, 0x2a, 0xc00051610e, 0xd, 0x0, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/src.go:102 +0x64d
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc0001e2800, 0xc0001df450)
	/Users/seer/git/github.com/containous/yaegi/interp/gta.go:146 +0xd48
github.com/containous/yaegi/interp.(*node).Walk(0xc0001e2800, 0xc0001df450, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:164 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc0001e2300, 0xc0001df450, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0001e2100, 0xc0001df450, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc00026e2c0, 0xc0001e2100, 0xc000036700, 0x1c, 0xc000036700, 0x1c, 0x4, 0x8, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/gta.go:18 +0x127
github.com/containous/yaegi/interp.(*Interpreter).importSrc(0xc00026e2c0, 0x0, 0x0, 0xc000188021, 0x1c, 0xc000188033, 0xa, 0xc000188021, 0xc00012f9c0)
	/Users/seer/git/github.com/containous/yaegi/interp/src.go:83 +0xdfa
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc000296300, 0x123bfe0)
	/Users/seer/git/github.com/containous/yaegi/interp/gta.go:146 +0xd48
github.com/containous/yaegi/interp.(*node).Walk(0xc000296300, 0xc0001dfdd8, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:164 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc000296200, 0xc0001dfdd8, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000296000, 0xc0001dfdd8, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:168 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc00026e2c0, 0xc000296000, 0xc0001800b8, 0x4, 0x0, 0xc0001800b8, 0x4, 0xc000296000, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/gta.go:18 +0x127
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc00026e2c0, 0xc000280030, 0x26, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:311 +0x105
github.com/containous/yaegi/interp.(*Interpreter).EvalWithContext.func1(0xc00027c180, 0xc00026e2c0, 0xc000280030, 0x26, 0xc00017c3a0, 0xc000178120)
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:396 +0x83
created by github.com/containous/yaegi/interp.(*Interpreter).EvalWithContext
	/Users/seer/git/github.com/containous/yaegi/interp/interp.go:394 +0x14d
```

--- a/_test/foo-bar/foo-bar.go
+++ b/_test/foo-bar/foo-bar.go
@@ -1,0 +1,3 @@
+package bar
+
+var Name = "foo-bar"

--- a/_test/import7.go
+++ b/_test/import7.go
@@ -1,0 +1,10 @@
+package main
+
+import "github.com/containous/yaegi/_test/foo-bar"
+
+func main() {
+	println(bar.Name)
+}
+
+// Output:
+// foo-bar

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -32,7 +32,7 @@ var constBltn = map[string]func(*node){
 	"real":    realConst,
 }
 
-var identifier = regexp.MustCompile("([\\pL_][\\pL_\\d]*)$")
+var identifier = regexp.MustCompile(`([\pL_][\pL_\d]*)$`)
 
 // cfg generates a control flow graph (CFG) from AST (wiring successors in AST)
 // and pre-compute frame sizes and indexes for all un-named (temporary) and named

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"log"
 	"math"
-	"path"
 	"reflect"
+	"regexp"
 	"unicode"
 )
 
@@ -31,6 +31,8 @@ var constBltn = map[string]func(*node){
 	"imag":    imagConst,
 	"real":    realConst,
 }
+
+var identifier = regexp.MustCompile("([\\pL_][\\pL_\\d]*)$")
 
 // cfg generates a control flow graph (CFG) from AST (wiring successors in AST)
 // and pre-compute frame sizes and indexes for all un-named (temporary) and named
@@ -316,7 +318,7 @@ func (interp *Interpreter) cfg(root *node) ([]*node, error) {
 				name = n.child[0].ident
 			} else {
 				ipath = n.child[0].rval.String()
-				name = path.Base(ipath)
+				name = identifier.FindString(ipath)
 			}
 			if interp.binPkg[ipath] != nil && name != "." {
 				sc.sym[name] = &symbol{kind: pkgSym, typ: &itype{cat: binPkgT, path: ipath}}

--- a/interp/gta.go
+++ b/interp/gta.go
@@ -1,7 +1,6 @@
 package interp
 
 import (
-	"path"
 	"reflect"
 )
 
@@ -126,7 +125,7 @@ func (interp *Interpreter) gta(root *node, rpath string) ([]*node, error) {
 				name = n.child[0].ident
 			} else {
 				ipath = n.child[0].rval.String()
-				name = path.Base(ipath)
+				name = identifier.FindString(ipath)
 			}
 			// Try to import a binary package first, or a source package
 			if interp.binPkg[ipath] != nil {

